DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/CourseSetupViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.Courses;
+
+    public class CourseSetupViewModel
+    {
+        public CourseSetupViewModel(IEnumerable<CourseStatistics> courses)
+        {
+            Courses = courses;
+        }
+
+        public IEnumerable<CourseStatistics> Courses { get; set; }

```

Comment:
If this is going to be a page with search/sort/filter in the future (like the wireframes suggest), this is going to need a separate view model rather than using the database entity. Doesn't necessarily need to change now, but it will likely need changing in the future